src/test/java/seedu/address/logic/commands/PolicyCommandTest.java

```diff
@@ -0,0 +1,125 @@
+        PolicyCommand policyCommand = new PolicyCommand(INDEX_FIRST_PERSON);
+
+        String expectedMessage = personPoliciesToDisplay.getPersonNameAndAllPolicies();
+
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        assertCommandSuccess(policyCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredPersonList().size() + 1);
+        PolicyCommand policyCommand = new PolicyCommand(outOfBoundIndex);
+
+        assertCommandFailure(policyCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_validIndexFilteredListNoPolicies_success() {
+        showPersonAtIndex(model, INDEX_THIRD_PERSON);
+
+        Person personPoliciesToDisplay = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        PolicyCommand policyCommand = new PolicyCommand(INDEX_FIRST_PERSON);
+
+        String expectedMessage = String.format(PolicyCommand.NO_POLICIES, personPoliciesToDisplay.getName());
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        // there should be no change to the model, since this is not a command which modifies the {@code model}
+        showPersonAtIndex(expectedModel, INDEX_THIRD_PERSON);
+
+        assertCommandSuccess(policyCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_validIndexFilteredListWithPolicies_success() {
+        showPersonAtIndex(model, INDEX_SECOND_PERSON);
+
+        Person personPoliciesToDisplay = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
+        PolicyCommand policyCommand = new PolicyCommand(INDEX_FIRST_PERSON);
+
+        String expectedMessage = personPoliciesToDisplay.getPersonNameAndAllPolicies();
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        // there should be no change to the model, since this is not a command which modifies the {@code model}
+        showPersonAtIndex(expectedModel, INDEX_SECOND_PERSON);
+
+        assertCommandSuccess(policyCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexFilteredList_throwsCommandException() {
+        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+
+        Index outOfBoundIndex = INDEX_SECOND_PERSON;
+        // ensures that outOfBoundIndex is still in bounds of address book list
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getPersonList().size());
+
+        PolicyCommand policyCommand = new PolicyCommand(outOfBoundIndex);
+
+        assertCommandFailure(policyCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+    }
```

Comment:
Each new line is meant to separate the logic from the other lines, so that each "section" gives me 1 variable/object that I would eventually pass into the assert function. I felt that it looks cleaner that way. What do you think?

Reply:
I see, I think I didn't notice the line breaks were separating each of the variables/objects. now it makes sense!